TYPO3.Neos/Resources/Private/Styles/_Global.scss

```diff
@@ -152,6 +152,11 @@ body {
 	}
 }
 
+// fix https://github.com/neos/neos-development-collection/issues/1387
+#neos-application {
+  position: fixed;
+}
```

Comment:
I highly recommend not doing this as it can have major implications on the various backend components.

E.g. the toolbar gets wrongly positioned when the "Edit / Preview" panel is opened and the content element toolbar disappears (wrongly positioned).

I wouldn't feel comfortable wanna give this +1 for anything but master and only after a lot of testing.

An alternative would be to add a new container that is fixed and use that as the container for tooltips instead of `#neos-application`. That should solve it without side-effects on other components.